packages/test/test-end-to-end-tests/src/test/summarizeRestart.spec.ts

```diff
@@ -168,22 +152,14 @@ describeNoCompat("Summarizer closes instead of refreshing", (getTestObjectProvid
 				);
 
 			await provider.ensureSynchronized();
-			let summary2Error: any;
-			summarizingContainer2.on("closed", (error) => {
-				summary2Error = error;
-			});
 
+			assert(!summarizingContainer2.closed, "Summarizer2 should not have closed yet!");
 			// This tells the summarizer to process the latest summary ack
 			// This is because the second summarizer is not the elected summarizer and thus the summaryManager does not
 			// tell the summarizer to process acks.
 			await summarizer2.run("test");
 
 			assert(summarizingContainer2.closed, "Unknown acks should close the summarizer");
-			assert.equal(
```

Comment:
Note with the change I made, I lose the ability to generate this error. Furthermore, I don't know if it actually stops the network call anymore.